pkg/kubernetes/helm.go

```diff
@@ -269,9 +269,9 @@ func (h *HelmExecute) RunHelmInstall() error {
 
 	if h.verbose {
 		helmParamsDryRun := helmParams
-		helmParamsDryRun = append(helmParamsDryRun, "--dry-run")
+		helmParamsDryRun = append(helmParamsDryRun, "--dry-run", "--hide-secret")
```

Comment:
we can remove the `helmParamsDryRun := helmParams` and just write it as follow
```suggestion
		helmParamsDryRun := append(helmParams, "--dry-run", "--hide-secret")
```